core/store/presenters/presenters.go

```diff
@@ -495,20 +495,25 @@ type Tx struct {
 }
 
 // NewTx builds a transaction presenter.
-func NewTx(tx *models.Tx) Tx {
+func NewTx(tx *models.Tx) (Tx, error) {
+	hex, err := utils.EncodeTxToHex(tx.EthTx(tx.GasPrice.ToInt()))
```

Comment:
Recapping an offline discussion, there is an issue here because we are generating the RLP hex of an `unsigned` ethereum transaction, while everywhere else we use a `signed` ethereum transaction. These are indeed different values. The good news is that this is confined to a presenter `hex` that doesn't get used outside of a visual.

```
unsignedHash:  0x38e4dc087a957b9e4463fb9c192112ba754d4cf5bfb3ce44838eb7f9eda7c60c
signedHash:  0x8852a363e64bbfcb91041cf99333fa901a37529f6ab3ebff4f952f1508beb90b
signedHash rlp encoded hex (desired):  0xf86a808504a817c8008307a120941ecff9a4d7a065d28ad8faf12b3489b9dd82cad480850000abcdef2aa09f4ca8c2e3378ffeb07c67ff6196f5a5c4db81dc4c9cb9cb52298ed5862bc9d4a0789b427d42cfc7dcad07eb90661f7d067b62975662bc4b7341b90ecd5084af8b
```

I'll commit a test demonstrating this. The failing in test in question will be `TestTx_PresenterMatchesHex`.